src/main/java/com/dev/cinema/dto/TicketResponseDto.java

```diff
@@ -0,0 +1,37 @@
+package com.dev.cinema.dto;
+
+public class TicketResponseDto {
+    private Long id;
+    private String movieTitle;
+    private Long userId;
```

Comment:
you already jave information about user in OrderResponseDto and ShoppingCartResponseDto. No need to add it to all TicketResponseDto(list of which is the part of both of above dtos)

Reply:
done